client/dashboard/me/mcp/setup/index.tsx

```diff
@@ -0,0 +1,437 @@
+												) }
+											</Text>
+										</li>
+									</ul>
+								</VStack>
+							</VStack>
+						</CardBody>
+					</Card>
+
+					<div style={ { marginTop: '24px' } }>
+						<Card>
+							<CardBody>
+								<VStack spacing={ 4 }>
+									<Text as="h1" size="large" weight={ 500 }>
+										{ __( 'MCP Client Configuration' ) }
+									</Text>
+									<SelectControl
+										__next40pxDefaultSize
+										__nextHasNoMarginBottom
+										label={ __( 'MCP Client' ) }
+										value={ selectedMcpClient }
+										options={ mcpClientOptions }
+										onChange={ setSelectedMcpClient }
+										help={ __( 'Choose your MCP client to get the correct configuration format.' ) }
+									/>
+								</VStack>
+							</CardBody>
+						</Card>
+					</div>
+
+					{ ( selectedMcpClient === 'claude' || selectedMcpClient === 'cursor' ) && (
+						<div style={ { marginTop: '24px' } }>
+							<Card>
+								<CardBody>
+									{ /* Quick Setup for Claude Desktop */ }
+									{ selectedMcpClient === 'claude' && (
+										<VStack spacing={ 4 }>
+											<Text as="h1" size="large" weight={ 500 }>
+												{ __( 'Quick Setup' ) }
+											</Text>
+											<Text as="p" size="medium" variant="muted">
+												{ __(
+													'For Claude Desktop users, we provide a one-click setup option using our MCP Bundle (MCPB) file.'
+												) }
+											</Text>
+											<Text as="p" size="medium" variant="muted">
+												{ __( 'Installation steps:' ) }
+											</Text>
+											<ol style={ { color: '#757575', paddingLeft: '20px', margin: '0' } }>
+												<li>
+													<Text as="p" size="medium" variant="muted">
+														{ __( 'Click the download button' ) }
+													</Text>
+												</li>
+												<li>
+													<Text as="p" size="medium" variant="muted">
+														{ __( 'Double-click the downloaded file' ) }
+													</Text>
+												</li>
+												<li>
+													<Text as="p" size="medium" variant="muted">
+														{ __( "Follow Claude Desktop's setup instructions" ) }
+													</Text>
+												</li>
+											</ol>
+											<Button
+												variant="primary"
+												href="https://github.com/Automattic/wpcom-mcp-bundle/raw/refs/heads/trunk/wordpress-com-mcp.mcpb"
+												target="_blank"
+												style={ { alignSelf: 'flex-start' } }
+											>
+												{ __( 'Download MCPB File' ) }
+											</Button>
+										</VStack>
+									) }
+
+									{ /* Quick Setup for Cursor */ }
+									{ selectedMcpClient === 'cursor' && (
+										<VStack spacing={ 4 }>
+											<Text as="h1" size={ 16 } weight={ 600 }>
+												{ __( 'Quick Setup' ) }
+											</Text>
+											<Text as="p" size="medium" variant="muted">
+												{ __(
+													'For Cursor users, we provide a one-click setup option that will automatically configure the MCP server.'
```

Comment:
Same. I think this is also the description of the SectionHeader, and the gap is 20px (`spacing={ 5 }`)